lib/ansible/executor/task_executor.py

```diff
@@ -393,7 +393,8 @@ def _run_loop(self, items):
             else:
                 if getattr(self._task, 'diff', False):
                     self._final_q.send_callback('v2_on_file_diff', tr)
-                self._final_q.send_callback('v2_runner_item_on_ok', tr)
+                if self._task.action not in ('add_host', 'group_by'):
```

Comment:
actually, add the 'fqcn' function and probably move this tuple to lib/ansible/constants.py